docs/readmes/howtos/troubleshooting/generate_admin_operator_certificates.md

```diff
@@ -0,0 +1,59 @@
+---
+id: generate_admin_operator_certificates
+title: Generate and update admin_operator certificates
+hide_title: true
+---
+# Generate and update admin_operator certificates
```

Comment:
I think at least some note that this same general process should be followed for docker as well can be helpful.

Reply:
@andreilee In this document, I think it is not too obvious what is the general process since we are using several terraform and kubernetes commands. We can try to make this more general process, however it may overcomplicate the steps with explanation. If there is demand of a step by step guide for docker, I can prepare a separate one, wdyt?

Reply:
Okay, let's leave it as is, then